### PR TITLE
Issue-3: Migration's database models created

### PR DESCRIPTION
**Pull request by LusineHovs**
Database models are added

### CODE REVIEW

**Comment by LusineHovs** [comment]
Squash must be used

**Review by martikyan — APPROVED** [review]

File: src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/Activity.cs [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+﻿using System;
+
+namespace Travely.ServiceManager.Abstraction.Models.Db
 {
     public class Activity
     {
         public long Id { get; set; }
+        public string Name { get; set; }
+        public string Address { get; set; }
+        public string ContactName { get; set; }
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/Activity.cs:10]: Please consider to use non-forgiving operator for similar properties.

File: src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/ActivityType.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Travely.ServiceManager.Abstraction.Models.Db
+{
+    public class ActivityType
+    {
+        public long Id { get; set; }
+        public string ActivityName { get; set; }
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/ActivityType.cs:9]: Name?

File: src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Travely.ServiceManager.Abstraction.Models.Db;
+
+namespace Travely.ServiceManager.DAL.Data
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs:4]: ".Data" can be removed

File: src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+{
+    public class ServiceManagerDbContext : DbContext
+    {
+        public ServiceManagerDbContext(DbContextOptions<ServiceManagerDbContext> options)
+            : base(options)
+        {
+        }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs:12]: New line above.

File: src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                .IsRequired();
+
+            modelBuilder.Entity<ActivityType>()
+                .Property(x => x.ActivityName)
+                .IsRequired();
+
+            base.OnModelCreating(modelBuilder);
+        }
+
+        public DbSet<Activity> Activities { get; set; }
```
Comment (tadart) [inline src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs:83]: Code parts with public access modifier should be placed higher than protected and private.

File: src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs (hunk 3 above)
Comment (tadart) [reply src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs:4]: The same for src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContextFactory.cs

File: src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/ActivityType.cs (hunk 2 above)
Comment (tadart) [reply src/ServiceManager/Travely.ServiceManager.Abstraction/Models/Db/ActivityType.cs:9]: As this field used as an index, better to define max length explicitly.

**Review by simonyangevorg — DISMISSED** [review]

**Review by tadart — APPROVED** [review]

**Review by tadart — APPROVED** [review]

**Review by simonyangevorg — APPROVED** [review]

**Pull request merged**